### DISCUSSION TITLE
Identity

### DISCUSSION THREAD

**Discussion by kofifus** [Dotnet-Boxed/Templates#203] [Q&A]
How would I add Identity (in my case Individual User Accounts) to the api template ?

Thanks!

**Comment by RehanSaeed** [comment]
Authorization is a complex subject with many different ways of implementing it. Including something out of the box that works for everyone is impossible. If you need `Individual User Accounts` auth (Microsoft specific), then create a project using Microsoft's default templates and copy the code across.
